app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -0,0 +1,47 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    # Generic JWT authenticator that receive JWT vendor configuration and uses to validate that the authentication
+    # request is valid, and return conjur authn token accordingly
+    JwtAuthenticate ||= CommandClass.new(
```

Comment:
can we call this `Authenticate`? this way the full name will be `Authentication::AuthnJwt::Authenticate`